docs/contribute/maintenance.rst

```diff
@@ -103,16 +103,16 @@ However, only people with ``Environments/Configure PyPI`` access can approve an
 
         git checkout main
         git pull
-        git checkout -b release main
-        git add CHANGES.rst
+        git checkout -b release-$VERSION main
```

Comment:
Oh, now I understand why you have that code block in Step 1!

Would you please add a sentence to Step 1 (I can't make a suggestion on that line in GitHub):

```rst
    Set an environment variable to use in subsequent commands during the release process.
```

Reply:
Perfect!